cupyx/time.py

```diff
@@ -0,0 +1,65 @@
+import time
+
+import numpy
+
+import cupy
+
+
+class _PerfCaseResult(object):
+    def __init__(self, name, ts):
+        assert ts.ndim == 2 and ts.shape[0] == 2 and ts.shape[1] > 0
+        self.name = name
+        self._ts = ts
+
+    @property
+    def cpu_times(self):
+        return self._ts[0]
+
+    @property
+    def gpu_times(self):
+        return self._ts[1]
+
+    @staticmethod
+    def _to_str_per_item(t):
+        assert t.size > 0
+        t *= 1e6
+
+        s = ' {:9.03f} us'.format(t.mean())
+        if t.size > 1:
+            s += '   +/-{:6.03f} (min:{:9.03f} / max:{:9.03f}) us'.format(
+                t.std(), t.min(), t.max())
+        return s
+
+    def to_str(self, show_gpu=False):
+        ts = self._ts if show_gpu else self._ts[[0]]
+        return '{:<20s}:{}'.format(
+            self.name, ' '.join([self._to_str_per_item(t) for t in ts]))
+
+    def __str__(self):
+        return self.to_str(show_gpu=True)
+
+
+def run(name, func, args=(), n=10000, *, n_warmup=10):
```

Comment:
I think most users are not interested in specifying `name`. How about making it keyword-only?

Reply:
Fixed.